vagrant/tutorial/post-reboot-bootstrap.py

```diff
@@ -102,10 +103,20 @@
 check_call(['truncate', '--size', '1G', '/var/opt/flocker/pool-vdev'])
 check_call(['zpool', 'create', 'flocker', '/var/opt/flocker/pool-vdev'])
 
+
+check_call(['mkdir', '/etc/flocker'])
+check_call(['cp', '/home/vagrant/credentials/cluster.crt',
+            '/etc/flocker/cluster.crt'])
+with file("/etc/flocker/agent.yml", "a") as f:
+    f.write(json.dumps({
+        "control-service": {"hostname": "172.16.255.250"},
+        "dataset": {"backend": "zfs"},
+        "version": 1
+    }))
+
 # Move SSH private key into place so ZFS agent can use it until we remove
```

Comment:
This logic is only run once, when building the base box, so we can't move the rest of the logic here.